third_party/istio/Makefile

```diff
@@ -0,0 +1,19 @@
+ROOT_DIR:=../..
+include $(ROOT_DIR)/Makefile.env
+include $(ROOT_DIR)/.mk/cluster.mk
+
+.PHONY: deploy
+deploy: $(TOOLBIN)/kubectl $(TOOLBIN)/istioctl
+	cd $(TOOLS_DIR); ./create_istio.sh
```

Comment:
Please update the installation to match the instructions in https://istio.io/latest/docs/setup/install/istioctl/

Reply:
Provide parameters to the installation using YAML file instead of flags

Reply:
ok